[ID] Update "Chat Console" (#5497)
ppy/osu-wiki: wiki/Chat_Console/id.md

# Konsol Chat

> Di dalam osu! kamu dapat menekan tombol `Show Chat` di pojok kanan bawah layar atau dengan menekan tombol `F8` untuk membuka konsol obrolan.

Reviewer: ```suggestion
Di dalam osu! kamu dapat menekan tombol `Show Chat` di pojok kanan bawah layar atau dengan menekan tombol `F8` pada keyboard untuk membuka konsol chat.
```

> ![Chat Console](img/Chatconsole1.png "Chat Console")

Reviewer: ```suggestion
![Tampilan Konsol Chat](img/Chatconsole1.png "Tampilan Konsol Chat")
```

> - Tab ini hanya ada beberapa channel. Cukup klik Tab yang lainnya untuk pindah ke channel tersebut atau dengan mengklik tanda tambah bewarna kuning `+` untuk membuka tab channel yang baru.

Reviewer: ```suggestion
- Daftar tab pada gambar tersedia beberapa kanal. Cukup klik tab lainnya untuk pindah ke kanal yang ingin dituju atau dengan mengklik tanda tambah berwarna kuning `+` untuk membuka tab kanal yang baru.
```

- Warna dari masing-masing pengguna memiliki arti yang berbeda.

| Warna | Siapa?  |
| :-- | :-- |
| Putih | Kamu |

> | Kuning Pucat | Orang Lain |

Reviewer: ```suggestion
| Kuning Pucat | Bukan pengguna osu!supporter |
```

> | Kuning | [osu!supporter](/wiki/osu!supporter) |

Reviewer: ```suggestion
| Kuning | Pengguna [osu!supporter](/wiki/osu!supporter) |
```

| Merah | Anggota [Tim Moderasi Global (GMT)](/wiki/People/The_Team/Global_Moderation_Team) atau [Tim Penjamin Kualitas (NAT)](/wiki/Glossary#quality-assurance-team) |

> | Hijau | Baris yang berisi nama pengguna anda atau berbagai kata kunci yang anda tambahkan ke daftar highlight. Salinan dari pesan itu juga akan muncul di channel yang terdedikasi untuk itu yaitu, `#highlight`. |

Reviewer: ```suggestion
| Hijau | Baris yang berisi nama pengguna kamu atau berbagai kata kunci yang ditambahkan ke daftar highlight. Salinan dari pesan itu juga akan muncul di kanal yang diperuntukkan untuk itu yaitu, `#highlight`. |
```

| Biru | Pesan Pribadi |
| Biru Terang | [peppy](https://osu.ppy.sh/users/2), pembuat osu! |
| Merah Muda | [BanchoBot](/wiki/BanchoBot) |

- Klik `Show Ticker` untuk menampilkan pesan obrolan terbaru di bagian bawah layar pada saat konsol chat tidak terlihat/terbuka.
- Klik `Auto-Hide` untuk menyembunykan konsol chat secara otomatis saat bermain (kecuali intro, outro dan jeda di beatmap).
- Klik `Hide Chat` atau tekan F8 lagi untuk menyembunyikan konsol chat.

## Konsol Chat yang Diperluas

*[osu!academy](/wiki/osu!academy) membuat dalam bentuk video [Episode 6 (6:52)](http://www.youtube.com/watch?v=cyYRl-a5xII) bersamaan dengan [Multiplayer](/wiki/Multi).*

Kamu dapat menekan `F9` atau klik tombol `Online Users` di bagian bawah kanan layar untuk membuka Konsol Chat yang Diperluas. Selain konsol obrolan, juga menampilkan informasi berbagai macam pengguna yang sedang daring osu!

> ![Extended chat console](img/Chat_Console-Extended.png "Extended chat console")

Reviewer: ```suggestion
![Chat konsol yang diperluas](img/Chat_Console-Extended.png "Chat konsol yang diperluas")
```

> Setiap pengguna yang masuk ke dalam osu! mempunyai panel pengguna yang ditampilkan konsol chat yang diperluas. Secara default, informasi umum pemain akan ditampilkan (nama, total skor ranked, peringkat, akurasi, jumlah bermain, dan avatar pengguna) pada panel pengguna. Jika mouse diarahkan ke panel pengguna, panel pengguna akan menampilkan informasi yang berbeda (nama, peringkat, avatar jika ada, waktu lokal mereka, lokasi negara dan kota, dan kegiatan yang sedang dilakukan oleh pengguna).

Reviewer: ```suggestion
Setiap pengguna yang masuk ke dalam osu! mempunyai panel pengguna yang ditampilkan pada konsol chat yang diperluas. Secara default, informasi umum pemain akan ditampilkan (nama, total skor ranked, peringkat, akurasi, jumlah bermain, dan avatar pengguna) pada panel pengguna. Jika mouse diarahkan ke panel pengguna, panel pengguna akan menampilkan informasi yang berbeda (nama, peringkat, avatar jika ada, waktu lokal mereka, lokasi negara dan kota, dan kegiatan yang sedang dilakukan oleh pengguna).
```

| `/bb` | Mengirim sebuah pesan ke bancho untuk rank `[nomor]` | `/bb rank 1` | \[15/11/12\] Stats for Uan: Score: 47,323,299,680 (#1) Plays: 176293 (lv102) Accuracy: 98.95% |
| `/chat [pengguna]` | Membuka sebuah tab obrolan baru dengan pengguna yang telah dispesifikasikan. | `/chat Amigo` | (Tab amigo akan terbuka) |
| `/clear` | Menghapus obrolan yang telah dimuat. (Hanya terhapus pada layarmu saja) | `/clear` | (Menghapus segalanya pada tab yang sedang kamu buka) |
| `/ignore [pengguna][@chp]` | Mengabaikan segala pesan dari pengguna yang telah dispesifikasikan pada sesi ini. Dengan menambahkan sebuah @ diikuti oleh huruf, c,h, dan/atau p, kamu mengabaikannya di obrolan, highlight, atau pesan pribadi secara berurut. | /ignore Amigo@chp | BanchoBot: You will no longer hear Amigo {chat} {highlights} {PM} (Konsol chat kamu disetel untuk: mengabaikan teks apapun yang Amigo tulis \[c\], teks yang mungkin menghighlightmu dari Amigo \[h\] Pesan pribadi yang dikirim Amigo kepadamu \[p\]) |

> | `/j [channel]` or `/join [channel]` | Memasuki channel yang dispesifikasikan | `/join #lobby` | (Tab #lobby akan terbuka) |

Reviewer: ```suggestion
| `/j [channel]` or `/join [channel]` | Bergabung ke spesifik kanal | `/join #lobby` | (Tab #lobby akan terbuka) |
```

> | `/p` or `/part` | Meninggalkan channel yang sedang kamu masuki (Perintah ini akan menutup jendela channel di tempat kamu mengetik perintah ini). | `/part` | n/a |

Reviewer: ```suggestion
| `/p` or `/part` | Meninggalkan kanal yang sedang kamu masuki atau meninggalkan kanal. | `/part` | n/a |
```
